### ISSUE TITLE
Documentation out of date

### ISSUE THREAD

**Issue by regispl** [Doist/todoist-python#20]
My app using `pytodoist` library stopped working today, so I came here to switch to the official client (it's my hobby project, so I didn't bother checking earlier if API is going to be deprecated :-) ). Unfortunately it seems like the doc is out of date as the `login` method is not present: 

```
In [3]: api = todoist.TodoistAPI()

In [4]: api.
api.activity             api.collaborators        api.items                api.queue                api.temp_ids
api.add_item             api.commit               api.labels               api.reminders            api.templates
api.api_endpoint         api.completed            api.live_notifications   api.reset_state          api.token
api.backups              api.deserialize          api.locations            api.serialize            api.uploads
api.biz_invitations      api.filters              api.notes                api.session              api.user
api.business_users       api.generate_uuid        api.project_notes        api.state
api.cache                api.get_api_url          api.projects             api.sync
api.collaborator_states  api.invitations          api.query                api.sync_token
```

**Comment by nosmo** [comment]
PR #19 addresses this issue.

**Comment by lefcha** [comment]
The `login` method has been relocated to the `api.user` manager in a recent refactoring that took place: http://todoist-python.readthedocs.io/en/latest/managers.html#todoist.managers.user.UserManager

So almost all methods that were residing in the main `api` module were also moved to the managers.

I will also update the `README` file.

**Comment by lefcha** [comment]
Closed by b5fe523971c0e8db727d6d7da9c5a3e561e5746c and aa5958f4a5c35246068a7668abdc83d0ea6c5bce.
